lib/src/matchers.rs

```diff
@@ -269,54 +269,67 @@ impl Matcher for IntersectionMatcher<'_> {
 /// visited.
 #[derive(PartialEq, Eq, Debug)]
 struct Dirs {
```

Comment:
It sounds like this PR makes the "Dirs" name a worse fit for this struct. Would something like `RepoPathTree` (or `...Trie`) make sense?

Reply:
`RepoPathTree` sounds good. Renamed, thanks.